docs/dev-guide.rst

```diff
@@ -206,26 +206,15 @@ Release notes
 =============
 
 To add entries to the release notes, create a file in the ``news`` directory
-with the ``source.type`` name format, where ``type`` is one of:
-
-* ``feature``: for new features
-* ``bug``: for bug fixes
-* ``api``: for API changes
-* ``dev``: for development-related changes
-* ``author``: for contributor names
-* ``other``: for other changes
-
-And where the ``source`` part of the filename is:
-
-* ``42`` when the change is described in issue ``42``
-* ``PR42`` when the change has been implemented in pull request ``42``, and
-  there is no associated issue
-* ``username`` for contributors (``author`` extention). It should be the
-  username part of their commit's email address.
-  
+with the ``id.type`` name format, where ``type`` is either label of the
```

Comment:
It's not clear from this descriptiong what label should be used. I would use something like `type is label of the issue that is named label.type (see supported types above)`. If the issue or PR is missing a label, please ask maintainer to add one.

Reply:
I thought `label` points to the `type`. Shouldn't be something like `id.label` instead of `label.type`?

Reply:
I wrote it backwards, it should be the name of the label `type.label`, so it's easy to recognize among other labels.